### PR TITLE
Codeql

### PR DESCRIPTION
**Pull request by Morriz**
Closes https://github.com/redkubes/unassigned-issues/issues/282.

### CODE REVIEW

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+    paths:
+      - src
```
Comment (j-zimnowoda) [inline .github/workflows/codeql-analysis.yml:30]: there is no src path in this repo.

File: .github/workflows/codeql-analysis.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    paths-ignore:
+      - src/node_modules
+      - '**/*.test.ts'
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: ['javascript']
```
Comment (j-zimnowoda) [inline .github/workflows/codeql-analysis.yml:38]: We do not have any javascript in otomi-core

**Review by j-zimnowoda — CHANGES_REQUESTED** [review]

File: .github/workflows/codeql-analysis.yml (hunk 2 above)
Comment (Dunky13) [reply .github/workflows/codeql-analysis.yml:38]: Not yet

File: .github/workflows/codeql-analysis.yml (hunk 1 above)
Comment (Dunky13) [reply .github/workflows/codeql-analysis.yml:30]: There will be with the new CLI

**Review by Dunky13 — APPROVED** [review]

**Comment by j-zimnowoda** [comment]
@Morriz I see there is one important change here. Please merge with master, so we can approve it.

**Comment by Morriz** [comment]
Not ready yet. Needs testing...

**Comment by svatwork** [comment]
Any update?

**Pull request closed without merging**